### PR TITLE
Move Baseline mention

### PR DESCRIPTION
**Pull request by martinthomson**
...and reword it slightly.

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Dec 3, 2024, 4:09 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Fw3ctag%2Fthe-web-is-not-versioned%2F156b937c156d24417fb487c9024139096f7869c4%2Findex.html%3FisPreview%3Dtrue)

```
Timed out after waiting 30000ms
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3ctag/the-web-is-not-versioned%237.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -85,15 +81,20 @@
 
 The web is a stable platform.
 Enhancements should always be incremental,
-in a positive direction
-(<a data-cite="design-principles#leave-the-web-better">leave the web better than you found it</a>),
+in a positive direction (leave the web better than you found it),
```
Comment (plinss) [inline index.html]: This is dropping Jeffrey's data-cite change

Comment (martinthomson) [reply index.html]:
```suggestion
in a positive direction (<a data-cite="design-principles#leave-the-web-better">leave the web better than you found it</a>),
```

File: index.html [hunk 2]
```diff
@@ -85,15 +81,20 @@
 
 The web is a stable platform.
 Enhancements should always be incremental,
-in a positive direction
-(<a data-cite="design-principles#leave-the-web-better">leave the web better than you found it</a>),
+in a positive direction (<a data-cite="design-principles#leave-the-web-better">leave the web better than you found it</a>),
```
Comment (martinthomson) [inline index.html]:
```suggestion
in a positive direction
(<a data-cite="design-principles#leave-the-web-better">leave the web better than you found it</a>),
```

File: index.html [hunk 3]
```diff
@@ -59,16 +59,12 @@
...
 so that we can iterate on them independently of each other.
-There are also efforts such as [Baseline](https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility),
-which provides the concept of an unversioned snapshot
-with interoperability guarantees,
-without assigning a version to the web platform as a whole.
 
 This allows the community to focus on the merits
 of each technical contribution to the web
-in a distributed and modular way.
+in a distributed and modular way.
```
Comment (martinthomson) [inline index.html]:
```suggestion
in a distributed and modular way.
```

**Review by rhiaro — APPROVED** [review]

**Pull request merged**